bikeshed/wpt/wptScript.js

```diff
@@ -44,6 +41,17 @@ document.addEventListener("DOMContentLoaded", async ()=>{
             })),
         );
         nameEl.insertAdjacentElement("afterend", resultsEl);
+
+	// Only update the summary pass/total count if we haven't seen this
+	// test before, to support authors listing the same test multiple times
+	// in a spec.
+        if (!seenTests.has(nameEl.getAttribute("title"))) {
```

Comment:
I know the indentation in this file is inconsistent between tabs and spaces (ugh, I dunno how that happened) but that doesn't mean that mixing 4-space and 2-space indents is allowed as well. ^_^ Mind switching it all to tab-based?